### PR TITLE
Speedup

### PR DESCRIPTION
**Pull request by khenderick**

### CODE REVIEW

File: webapps/api/backend/decorators.py [hunk 1]
```diff
@@ -81,7 +82,11 @@ def new_function(*args, **kw):
             if not ApiToolbox.is_token_in_roles(request.token, roles):
                 raise HttpForbiddenException(error_description='This call requires roles: {0}'.format(', '.join(roles)),
                                              error='invalid_roles')
-            return f(*args, **kw)
+            duration = time.time() - start
```
Comment (kvanhijf) [inline webapps/api/backend/decorators.py:85]: Shouldn't the duration be calculated after having executed the function?

File: webapps/api/backend/decorators.py [hunk 2]
```diff
@@ -215,8 +221,12 @@ def new_function(*args, **kwargs):
             # Execute validator
             if validator is not None:
                 validator(args[0], **validation_new_kwargs)
+            duration = time.time() - start
             # Call the function
-            return f(args[0], **new_kwargs)
+            result = f(args[0], **new_kwargs)
```
Comment (kvanhijf) [inline webapps/api/backend/decorators.py:226]: Same remark

File: webapps/api/helpers.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+"""
+Some helpers
+"""
+
+from rest_framework.response import Response
+
+
+class OVSResponse(Response):
+
+    def __init__(self, data=None, status=200,
```
Comment (kvanhijf) [inline webapps/api/helpers.py:27]: Using a generic approach here, seems better to me
`super(OVSResponse, self).__init__(**kwargs)`

**Review by kvanhijf — CHANGES_REQUESTED** [review]

File: webapps/api/backend/decorators.py (hunk 1 above)
Comment (khenderick) [reply webapps/api/backend/decorators.py:85]: No, as the goal is to time how long the security checking takes, not how long security checking *and* executing the function takes.

File: webapps/api/helpers.py (hunk 3 above)
Comment (khenderick) [reply webapps/api/helpers.py:27]: Would be a bit more awkward since the `timings` arg is at the end, so I'd have to find it in `kwargs` or `args`.

**Review by kvanhijf — APPROVED** [review]

**Pull request merged**